core/uploadDiveLogsDE.cpp

```diff
@@ -11,6 +11,9 @@
 #include "core/membuffer.h"
```

Comment:
comment on the commit message for commit a4db3d0162dae2ec2d
the subject line is way too long, the line number references feel odd to me, simply go through it in order and the reader can easily figure it out
Other than that, I think is the right thing to do.

Reply:
Corrected

Reply:
Corrected